File: Unidad 2-Estructuras de Control/Ejercicio15.java
//Equipo 3
//Diego De Gante
//Programa que lea dos fechas (día-mes-año) con tres cantidades enteras 
//e imprima la mayor, si son la misma imprima son iguales.
//Entrada:diaUno, diaDos, mesUno, mesDos, anioUno, anioDos
//Salida: La fecha mayor o si las fechas son iguales

import java.util.Scanner;

public class Main{
 public static void main(final String arg[]) {

   // Definir variables
   final Scanner entrada = new Scanner(System.in);
   int diaUno = 0, mesUno = 0, anioUno = 0, diaDos = 0, mesDos = 0, anioDos = 0;

   //Solicita y agarra el dato ingresado por el usuario para fecha 1
   System.out.println("Ingrese la 1er fecha");
   System.out.println("Ingrese el día(1-31)");
   diaUno = entrada.nextInt();
   System.out.println("Ingrese el mes(1-12)");
   mesUno = entrada.nextInt();
   System.out.println("Ingrese el año");
   anioUno = entrada.nextInt();

   //Solicita y agarra el dato ingresado por el usuario para fecha 2
   System.out.println("Ingrese la 2da fecha");
   System.out.println("Ingrese el día(1-31)");
   diaDos = entrada.nextInt();
   System.out.println("Ingrese el mes(1-12)");
   mesDos = entrada.nextInt();
   System.out.println("Ingrese el año");
   anioDos = entrada.nextInt();

   //Comprobacion de fecha por orden(año, mes, día)
   if(anioUno > anioDos){
       System.out.println(diaUno); 
       System.out.println(mesUno);
       System.out.println(anioUno);
   } else if(anioDos > anioUno){
      System.out.println(diaDos);
      System.out.println(mesDos);
      System.out.println(anioDos);;
   } else  if(anioUno == anioDos){
       if(mesUno > mesDos){
          System.out.println(diaUno); 
          System.out.println(mesUno);
          System.out.println(anioUno);
       } else if(mesDos > mesUno){
          System.out.println(diaDos);
          System.out.println(mesDos);
          System.out.println(anioDos);
       } else if(mesUno == mesDos){
           if(diaUno > diaDos){
              System.out.println(diaUno); 
              System.out.println(mesUno);
              System.out.println(anioUno);
           } else if (diaDos > diaUno){
               System.out.println(diaDos);
               System.out.println(mesDos);
               System.out.println(anioDos);
           } else if (diaUno == diaDos){
               System.out.println("Las fechas son iguales");
           }
       }
   }
 }
}

File: Unidad 2-Estructuras de Control/Ejercicio15.c
//Equipo 3
//Diego De Gante
//Programa que lea dos fechas (día-mes-año) con tres cantidades enteras 
//e imprima la mayor, si son la misma imprima son iguales.
//Entrada:diaUno, diaDos, mesUno, mesDos, anioUno, anioDos
//Salida: La fecha mayor o si las fechas son iguales

#include <stdio.h>

int main()
{
    int diaUno = 0, mesUno = 0, anioUno = 0, diaDos = 0, mesDos = 0, anioDos = 0;;
    printf("1ra Fecha\n");
    printf("Ingresa el número del día\n");
    scanf("%i", &diaUno);
    printf("Ingresa el número del mes\n");
    scanf("%i", &mesUno);
    printf("Ingresa el número del año\n");
    scanf("%i", &anioUno);

    printf("2da Fecha\n");
    printf("Ingresa el número del día\n");
    scanf("%i", &diaDos);
    printf("Ingresa el número del mes\n");
    scanf("%i", &mesDos);
    printf("Ingresa el número del año\n");
    scanf("%i", &anioDos);

    if (anioUno > anioDos)
    {
        printf("%i\n",diaUno);
        printf("%i\n",mesUno);
        printf("%i\n",anioUno);

    }else if(anioUno < anioDos)
    {
        printf("%i\n",diaDos);
        printf("%i\n",mesDos);
        printf("%i\n",anioDos);

    }else
    {
        if (mesUno > mesDos)
        {
            printf("%i\n",diaUno);
            printf("%i\n",mesUno);
            printf("%i\n",anioUno);

        }else if (mesUno < mesDos)
        {
            printf("%i\n",diaDos);
            printf("%i\n",mesDos);
            printf("%i\n",anioDos);

        }else
        {
            if (diaUno > diaDos)
            {
                printf("%i\n",diaUno);
                printf("%i\n",mesUno);
                printf("%i\n",anioUno);

            }else if (diaUno < diaDos)
            {
                printf("%i\n",diaDos);
                printf("%i\n",mesDos);
                printf("%i\n",anioDos);

            }else
            {
                printf("Las fechas son iguales\n");
            }
        }
    }
    return 0;
}